tests/backend/PortabilityService.Gateway.Tests/AspNetCoreTestHelpers/WebHostHelper.cs

```diff
@@ -15,12 +19,23 @@ internal static class TestGateway
     {
         internal static TestServer CreateTestApp()
         {
-            var webHostBuilder = Program.CreateWebHost(null);
+            var webHostBuilder = CreateTestWebHost();         
 
             // Set the content root so that correct config files (reroutes.json) are loaded
             webHostBuilder.UseContentRoot(Path.GetDirectoryName(typeof(TestGateway).Assembly.Location));
 
             return new TestServer(webHostBuilder);
         }
+
+        private static IWebHostBuilder CreateTestWebHost() =>
+            WebHost.CreateDefaultBuilder()
+            .ConfigureAppConfiguration(ConfigureTestAppConfiguration)
+            .UseStartup<Startup>();
+
+        private static void ConfigureTestAppConfiguration(WebHostBuilderContext context, IConfigurationBuilder configBuilder)
+        {
+            configBuilder.AddJsonFile("reroutes.json");
+            configBuilder.AddPortabilityServiceConfiguration(new Uri("http://localhost"), ConfigurationProviderConstants.PortabilityServiceConfigurationRoot, true);
```

Comment:
Should we maybe just leave this out for test scenarios since it was causing problems and we don't need configuration from the service to test the gateway?

Reply:
Removed